app/iht/utils/tnrb/TnrbHelper.scala

```diff
@@ -57,6 +57,19 @@ object TnrbHelper {
     }
   }
 
+  /**
+    * Returns the Spouse name (if exists) otherwise returns the pretext string
+    */
+  def spouseOrCivilPartnerName(tnrbModel: TnrbEligibiltyModel,
+                                prefixText: String=""): String  = {
+    if(tnrbModel.Name.toString.trim!=""){
```

Comment:
More efficient and readable version:
    CommonHelper.withValue(tnrbModel.Name.toString.trim) {
      case name if name.isEmpty => prefixText
      case name => name
    }